ui/components/src/FormatControls/FormatControls.tsx

```diff
@@ -61,7 +66,7 @@ export function FormatControls({ value, onChange, disabled = false }: FormatCont
 
   const handleKindChange = (_: unknown, newValue: AutocompleteUnitOption): void => {
     onChange({
-      unit: newValue.id,
+      unit: newValue.id ?? 'decimal', // Fallback to 'decimal' if no unit is selected
```

Comment:
This will mean that when we try to edit a dashboard that was created with go, then we will always change it to decimal. Maybe what we need to support here is to be able to select and undefined/empty value to the unit. This will match what the SDK does.